### DISCUSSION TITLE
How to install ATSAS on clusters

### DISCUSSION THREAD

**Discussion by smallfishabc** [biosaxs-com/atsas-community#1955] [ATSAS package in general]
Hi,
I want to use RAW API and ATSAS on the computing cluster. Our cluster has the Rocky Linux 8. Could you tell me how to install it?
Do you have any source code for compiling?
Best
Feng Yu

**Comment by franke-biosaxs** [comment]
You can try the package for OracleLinux-8; OracleLinux and RockyLinux have the same RedHat based, as far as I know. No guarantees there, though. Such an install is not supported.
